### DISCUSSION TITLE
Multiple identities

### DISCUSSION THREAD

**Discussion by omgitsraven** [beakerbrowser/beaker#1870] [Q&A]
I'm trying to create a web app where each user will have their own personal files, but can also view other users' files (read-only) in the same app.

Is there any way to create a second identity, with separate permissions, in (for example) a second Beaker window, so that I can test viewing another user's files without having to have a second physical machine to create the "other user"'s files on?

(I'm assuming that creating a new Windows user account might let me do it from a single machine, but switching back and forth between two separate Windows user accounts is only slightly less inconvenient at that...)

**Comment by pfrazee** [comment]
Unfortunately I think you'll just need to "fake it" for now. I believe the only state you need to modify to fake it is the `writable` flag so it's not terribly difficult.
